GameEngine/Characters/PlayerCharacter.cs

```diff
@@ -13,8 +13,12 @@ public PlayerCharacter(string name, int hp) : base(name, hp)
 
         internal void InternalTurn(EngineInternal engine)
         {
+            Console.Clear();
```

Comment:
Let's not put in any clears unless the player uses the clear command explicitly.

Reply:
It makes sense for it to be there, I tested it, you did not.